CV/assets/css/style.css

```diff
@@ -0,0 +1,110 @@
+body {
+    font-family: 'RobotoLight', Helvetica, Arial, sans-serif;
+    margin: 10px 30px 0 30px;
+}
+
+p {
+    font-size: 20px;
+}
+
+.center {
+    margin: 0 auto;
+}
+
+.masthead {
+    height: 80px;
+    text-align: center;
+    padding: 20px 0;
+    background: linear-gradient(#e66465, #9198e5);
+
+}
+
+.info {
+    display: flex;
+    justify-content: space-around;
+    margin: 10px 0;
+    width: 100%;
+}
+
+.skills {
+    position: relative;
+    padding: 20px 20px;
+    background: linear-gradient(#e66465, #9198e5);
+    max-width: 30%;
+    min-width: 15%;
+    margin: 0 10px 0 0;
+}
+
+.description {
+    position: relative;

```

Comment:
What the meaning of using relative positioning?

Reply:
In the process of creating, I thought to use this positioning as a container for the absolute, but in the end I abandoned this idea because it was unnecessary (I did not use absolute positioning) and left the relative just because, judging by the articles on the Internet, this does not affect performance.